### PR TITLE
docs: fix strictTemplates for angular 9+

### PR DESCRIPTION
**Pull request by johnpapa**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

doc shows previous tsconfig setting, and should show `strictTemplates`

## What is the new behavior?

doc will show tsconfig setting `strictTemplates`

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

cc @IgorMinar

### CODE REVIEW

File: aio/content/guide/typescript-configuration.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ The initial `tsconfig.json` for an Angular app typically looks like the followin
   },
   "angularCompilerOptions": {
     "fullTemplateTypeCheck": true,
-    "strictInjectionParameters": true
+    "strictTemplates": true
```
Comment (IgorMinar) [inline aio/content/guide/typescript-configuration.md]: oops.. you changed the wrong line....

```suggestion
    "strictTemplates": true,
    "strictInjectionParameters": true
```

File: aio/content/guide/typescript-configuration.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ The initial `tsconfig.json` for an Angular app typically looks like the followin
   },
   "angularCompilerOptions": {
     "fullTemplateTypeCheck": true,
```
Comment (IgorMinar) [inline aio/content/guide/typescript-configuration.md]:
```suggestion
```

**Review by IgorMinar — CHANGES_REQUESTED** [review]
please also update the commit message with the reasoning for this change the "why" for this change.

File: aio/content/guide/typescript-configuration.md (hunk 2 above)
Comment (johnpapa) [reply aio/content/guide/typescript-configuration.md]: docs: intended to add strictTemplates in place of fullTemplateTypeCheck

**Comment by IgorMinar** [comment]
can you please squash this into a single commit?

**Comment by johnpapa** [comment]
@googlebot I fixed it

**Review by IgorMinar — APPROVED** [review]
I've rebased this and fix the last remaining ws issues. This now looks good to me. Thanks @johnpapa!

I've relabeled this for master-only, as the md file describes the current default which still is `fullTemplateTypecheck` on not `strictTemplates`, but since we are changing this in v10, we can go ahead and merge this in, so that it's available on next.angular.io.

// fyi @clydin

**Pull request closed without merging**